draft-jholland-quic-multicast.md

```diff
@@ -737,6 +737,14 @@ A client might receive multicast packets that it can not associate with any chan
 This traffic is presumed either to have been corrupted in transit or to have been sent by someone other than the legitimate sender of traffic for the channel, possibly by an attacker or a misconfigured sender.
 If these packets are addressed to an (S,G) that is used for reception in one or more known channels, the client MAY leave these channels with reason "Excessive Spurious traffic".
 
+## Retransmission of information
+In addition to the mechanisms used for retransmission described in {{Section 13.3 of RFC9000}} and {{Section 5.2 of RFC9221}} the following rules apply to the newly introduced frames:
+
+- As the properties carried in MC_ANNOUNCE frames can not change during the lifetime of a channel, information contained in them can be retransmitted one to one.
+- Since conditions of the client or channel can have changed by the time a retransmission of an MC_JOIN, MC_LEAVE or MC_RETIRE channel becomes necessary, a retransmission might no longer be required or even appropriate. A retransmission MUST only occur if the channel in question should still be joined/left/retired.
```

Comment:
I don't think this MUST is required and I think it complicates the handling for the client trying to send updates unnecessarily.

But we do need to explain that only the latest MC_STATE sequence number should be accepted in the MC_STATE handling section, which it looks like is omitted currently, so that if multiple MC_STATE messages are sent reliably the server will converge on the correct understanding.

Also, if we do this "MUST only occur when" I think can be phrased as "MUST NOT occur unless".  I don't think 2119/8174 would give a coherent understanding for "MUST only occur when".